.github/workflows/test-dbt-installation-pip.yml

```diff
@@ -88,9 +92,14 @@ jobs:
         run: |
           python -m pip show ${{ inputs.package_name }}
 
+      - name: "Verify dbt-core Version"
+        if: ${{ inputs.package_name == 'dbt-core' }}
+        run: dbt --version
```

Comment:
Any reason to have this instead of           python -c "import dbt" or some such.

Reply:
We could do that, I was just defaulting to the original check. I also like that we're actually running dbt, and not just checking that the package is installable. If I could think of a good alternative for the adapters, I'd probably update that to something like this as well.